web_ui/ui.go

```diff
@@ -597,11 +597,12 @@ func configureCommonEndpoints(engine *gin.Engine) error {
 
 	downtimeAPI := engine.Group("/api/v1.0/downtime")
 	{
-		downtimeAPI.POST("", AuthHandler, AdminAuthHandler, HandleCreateDowntime)
+		downtimeAPI.POST("", DowntimeAuthHandler, HandleCreateDowntime)
+		downtimeAPI.POST("/:uuid", DowntimeAuthHandler, HandleCreateDowntime)
```

Comment:
Why this endpoint? How would I know the id of the downtime before it is created?